src/pages/AccountSettings/tabs/Profile/ActivationBanner/useSelfActivationMutation.js

```diff
@@ -18,43 +21,51 @@ export const useSelfActivationMutation = ({ queryClient, canChange }) => {
     },
     onMutate: async (activated) => {
       await queryClient.cancelQueries(['SelfHostedCurrentUser'])
-      await queryClient.cancelQueries(['Seats'])
+      await queryClientV5.cancelQueries({
+        queryKey: SelfHostedSeatsConfigQueryOpts({ provider }).queryKey,
+      })
 
       const prevUser = queryClient.getQueryData(['SelfHostedCurrentUser'])
-      const prevSeat = queryClient.getQueryData(['Seats'])
+      const prevSeat = queryClientV5.getQueryData(
+        SelfHostedSeatsConfigQueryOpts({ provider }).queryKey
+      )
 
       if (canChange) {
         queryClient.setQueryData(['SelfHostedCurrentUser'], (user) => ({
           ...user,
           activated,
         }))
 
-        queryClient.setQueryData(['Seats'], (seats) => {
-          const seatsUsed = seats?.data?.config?.seatsUsed
-
-          return {
-            data: {
-              config: {
-                ...seats?.data?.config,
-                seatsUsed: activated ? seatsUsed + 1 : seatsUsed - 1,
+        queryClientV5.setQueryData(
+          SelfHostedSeatsConfigQueryOpts({ provider }).queryKey,
+          (seats) => {
+            const seatsUsed = seats?.data?.config?.seatsUsed
+            return {
+              data: {
+                config: {
+                  ...seats?.data?.config,
+                  seatsUsed: activated ? seatsUsed + 1 : seatsUsed - 1,
+                },
               },
-            },
+            }
           }
-        })
+        )
       }
 
-      return {
-        prevUser,
-        prevSeat,
-      }
+      return { prevUser, prevSeat }
     },
     onError: (_err, _activated, context) => {
       queryClient.setQueryData(['SelfHostedCurrentUser'], context.prevUser)
-      queryClient.setQueryData(['Seats'], context.prevSeat)
+      queryClientV5.setQueryData(
+        SelfHostedSeatsConfigQueryOpts({ provider }).queryKey,
+        context.prevSeat
+      )
     },
     onSettled: () => {
       queryClient.invalidateQueries(['SelfHostedCurrentUser'])
-      queryClient.invalidateQueries(['Seats'])
+      queryClientV5.invalidateQueries(
+        SelfHostedSeatsConfigQueryOpts({ provider }).queryKey
```

Comment:
just making sure - is this provider key used in the lookup case-sensitive at all? Just in case that misses hits between like GITHUB, github, .. Oh and I guess also the shorthand like gh.

Reply:
yes it would be case sensitive, however the data fetching query key is also case sensitive so they would align on that level and ensure that the correct value in the cache gets cleared.